framework_agreement_department/model/portfolio.py

```diff
@@ -0,0 +1,36 @@
+# Author: Leonardo Pistone
+# Copyright 2015 Camptocamp SA (http://www.camptocamp.com)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public Lice
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from openerp import models, fields
+
+
+class Portfolio(models.Model):
+    _inherit = 'framework.agreement.portfolio'
+
+    def _get_my_department(self):
```

Comment:
Not something I'll insist on.
On general note, IMO compatibility with old API shuuld be of worry only for still existing OCA code. I see no issue on requiring future code to be new API.